.github/workflows/main.yml

```diff
@@ -0,0 +1,132 @@
+name: Tbots CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  formatting-check:
+    runs-on: ubuntu-18.04
+    steps:
+      # checks-out the repository under $GITHUB_WORKSPACE
+      - uses: actions/checkout@v2
+        
+      - name: Environment Setup
+        run: |
+          "${GITHUB_WORKSPACE}"/environment_setup/setup_software.sh
+      
+      - name: Check Formatting
+        run: |
+          ./formatting_scripts/check_formatting_ci.sh
```

Comment:
Instead of checkFormatting, could we instead add fixFformatting as one of the scripts it runs?

Reply:
We generally do not want CI changing our code and committing it, because for some odd reason if it messes something up, we won't know what went wrong. Its really up to the dev to run fix formatting and CI will just check and make sure that the `fix_formatting.sh` script was run.